newsubprojects.md

```diff
@@ -0,0 +1,183 @@
+  technologies against a backdrop of the broader project design efforts.
+* Allow the community and Steering Council to re-evaluate existing Subprojects in the main
+  jupyter GitHub organizations to determine if any need to be moved to jupyter-incubator.
+* Allow the community to make sure that Subprojects develop into something that is
+  appropriate to become an officially supported and maintained part of the project.
+
+### Proposal for incubation
+
+A Subproject team can initiate the [jupyter-incubator](https://github.com/jupyter-incubator)
+incubation process by submitting a proposal for incubation. This process is designed to be
+lightweight and quick:
+
+1. The proposers must fill out incubation application by sumitting a pull request to the
+   [jupyter-incubator/proposals](https://github.com/jupyter-incubator/proposals) repository.
+2. Proposers must announce their intent to the community with a post to the main Jupyter
+   list.
+2. The proposed Subproject must have an Advocate who is an active Steering Council member.
+3. The proposal for incubation will be discussed by the community and approved or rejected by
+   the consensus of the Steering Council.
+4. If approved, the proposal pull request will be merged, if rejected, it will be closed.
+
+### Incubation period
+
+Once a Subproject is approved for incubation in the
+[jupyter-incubator](https://github.com/jupyter-incubator) organization, the following steps will be
+taken:
+
+
+1. A GitHub repo will be created under the jupyter-incubator organization
+2. A GitHub team will be created with read/write access to the repo, including the
+   Advocate.
+3. The standard Jupyter LICENSE file will be added to the repository.
+4. A member of the Steering Council will announce the newly incubated Subproject on the
+   main Jupyter list.
+
+The goal of the incubation period is to demonstrate that the Subproject is going to develop an
+with an active developer and user community. There is no particular length of time required,
+but most projects will be in incubation for at least 6 months to 1 year.
+
+## External incubation
+
+Incubation outside the [jupyter-incubator](https://github.com/jupyter-incubator) organization is
+also possible. In this case, there is no process. Any individual or organization can simply create
+a new project on their own personal GitHub (or other VCS) repository and develop it as they see
+fit. This is a very common pattern in the broader open source community; someone starts out
+developing something in their own personal GitHub organization, and later, after the project
+matures, it is transfered and incorporated into a larger GitHub organization.
+
+If an externally created and incubated subproject wants to become part of the official Jupyter
+organization, the criteria listed at the top of this document must be satisfied and the same
+incorporation process must be followed.
```

Comment:
I agree with @jasongrout, this last section is important. I would suggest we add a intra-document link back to the header for the section associated with "criteria listed at the top of this document".